website/addons/wiki/utils.py

```diff
@@ -154,3 +155,64 @@ def format_wiki_version(version, num_versions, allow_preview):
         raise InvalidVersionError
 
     return version
+
+def serialize_wiki_settings(user, node_ids):
```

Comment:
Why pass a list of node IDs? In its only usage, this function takes only a single node ID. Why not just pass a single node object?